Userland/Games/Hearts/SettingsDialog.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "SettingsDialog.h"
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/TextBox.h>
+
+SettingsDialog::SettingsDialog(GUI::Window* parent, String player_name)
+    : GUI::Dialog(parent)
+    , m_player_name(player_name)
+{
+    set_rect({ 0, 0, 250, 100 });
```

Comment:
```suggestion
    set_rect({ 0, 0, 250, 75 });
```

I think it looks slightly better with less horizontal space between the buttons. Maybe give this a try. :)